packages/nimble-components/package.json

```diff
@@ -44,6 +40,53 @@
     "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
     "test": "npm run test-chrome:verbose && npm run test-firefox:verbose"
   },
+  "wireit": {
```

Comment:
Some of these steps likely need to be triggered if `package.json` or `node_modules` changes.

Reply:
As Milan mentioned, Wireit defaults to picking up changes to `package-lock.json`, so we don't have to worry.